libs/data-types/src/test/custom-data-schema.spec.ts

```diff
@@ -46,11 +47,14 @@ describe('NonEmptyString', () => {
         @NestedDataType(Whatsit, { isOptional: true })
         secondaryWhatsit = {};
 
-        @RawData()
-        rawData = { foo: 72 };
+        @RawDataObject()
+        rawDataObject = { foo: 72 };
 
-        @RawData({ isOptional: true })
+        @RawDataObject({ isOptional: true })
         optionalRawData = undefined;
+
+        @Enum(CoscradEnum.MIMEType)
```

Comment:
I'll be interested to learn more about this...

Reply:
Discussed this at standup.